app/src/main/java/vn/mbm/phimp/me/editor/editimage/EditImageActivity.java

```diff
@@ -224,6 +234,21 @@ public void onFling(MotionEvent e1, MotionEvent e2, float velocityX, float veloc
         });
     }
 
+    private void shareImage() {
+        if(mOpTimes>0) {
```

Comment:
Why this check? It force user to edit image at least once to enable the share button. User can share any image edited or not.